doc/source/management/data_at_rest_encryption.rst

```diff
@@ -454,6 +454,29 @@ Dumping of encrypted binary logs involves decryption, and can be done using
    relevant to binary logs, encrypted relay logs can not be dumped/decrypted
    in this way.
 
+.. rubric:: Upgrading from |Percona Server| |changed-version| to any higher version
+
+The key format in the :ref:`keyring vault plugin
+<keyring_vault_plugin>` was changed for binlog encryption in |Percona
```

Comment:
keyring_vault?

Reply:
Github, like many diff programs, uses a line-by-line comparison which is not a great idea for documentation: lines don't matter in documentation, statements and sentences do. This sentence here was not changed but the text width was aligned to 80 characters. Git (and Github) produced a false positive here.
What changed is the following:
When you are upgrading from |Percona Server| 5.7.20-19 to a higher
version in the |Percona Server| 5.7 series or to a version prior to 8.0.15-5 in
the |Percona Server| 8.0 series, the binary log encryption will work after you
complete the following steps
I also added this reference:
.. seealso::

   |Percona Server| Documentation: Important changes in |Percona Server| 8.0.15-5
      - `Binary log encryption to use the upstream implementation
	<https://www.percona.com/doc/percona-server/LATEST/management/data_at_rest_encryption.html#binary-log-encryption>`_

Reply:
But github does not show old text version, with presumably unchanged sentence, on the left. Perhaps rebase the PR?

Reply:
Clarification: The format change is mentioned because of the update related to https://jira.percona.com/browse/PS-5207 (document that upgrade from 5.7.20-19 with binlog encryption to any newer version requires additional steps). As I see, my documentation update then did not affect 5.7 (only 8.0). As a result, the documentation update suggested in this pull request combines the solution of the problem for PS-5430 and PS-5207.